packages/load/tests/loaders/documents/documents-from-glob.spec.ts

```diff
@@ -93,15 +93,12 @@ describe('documentsFromGlob', () => {
     });
 
     test(`Should throw on empty files and empty result`, async () => {
-      try {
-        const glob = join(__dirname, 'test-files', '*.empty.graphql');
-        await load(glob, {
-          loaders: [new GraphQLFileLoader()],
-        });
-        expect(true).toBeFalsy();
```

Comment:
Not only related to this test but I found `expect(true).toBeFalsy();` will actually throw an error, which will satisfy the `expect(e).toBeDefined();` statement below 
i.e. the tests in this file will always pass 😅